### PR TITLE
Adding the additional env vars for provisioning setup

### PR DESCRIPTION
**Pull request by adarshdubey-star**

### CODE REVIEW

File: pipelines/basic.yaml [hunk 1]
```diff
@@ -269,6 +269,8 @@ spec:
         - name: IQE_ENV
           value: "$(params.IQE_ENV)"
         - name: IQE_ENV_VARS
+          value: "$(params.IQE_ENV_VARS)"
+        - name: IQE_ADDITIONAL_ENV_VARS
           value: "$(tasks.custom-preparation.results.iqe-additional-env-vars)"
```
Comment (adarshdubey-star) [inline pipelines/basic.yaml:274]: I have added the task reference here for the new var

File: tasks/run-iqe-cji.yaml [hunk 2]
```diff
@@ -137,6 +143,10 @@ spec:
         #!/bin/bash
         set -ex
 
+        if [ -z "$IQE_ENV_VARS" ]; then
```
Comment (adarshdubey-star) [inline tasks/run-iqe-cji.yaml:146]: This checks if the IQE_ENV_VARS are empty we could pass the new values

Comment (adarshdubey-star) [reply tasks/run-iqe-cji.yaml:146]: we are doing something similar here for the empty vars - https://github.com/RHEnVision/bonfire-tekton/blob/main/tasks/deploy.yaml#L112

File: tasks/run-iqe-cji.yaml [hunk 3]
```diff
@@ -137,6 +143,10 @@ spec:
...
+          export IQE_ENV_VARS="$IQE_ADDITIONAL_ENV_VARS"
```
Comment (adarshdubey-star) [inline tasks/run-iqe-cji.yaml:147]: I don't think we need to export right ?

File: tasks/run-iqe-cji.yaml (hunk 2 above)
Comment (ezr-ondrej) [reply tasks/run-iqe-cji.yaml:146]: That's actually a good solution, tho that case a nit:
We could name it `IQE_ENV_VARS_SUBSTITUDE`, but that's really just a nit, if we have a good description, I don't mind either way :)

File: tasks/run-iqe-cji.yaml [hunk 4]
```diff
@@ -59,6 +59,10 @@ spec:
       type: string
       description: "Comma delimited string with additional values, such as IQE_ENV_VARS='DYNACONF_USER_PROVIDER__rbac_enabled=false,foo=bar'"
       default: ""
+    - name: IQE_ADDITIONAL_ENV_VARS
+      type: string
+      description: "Comma delimited string with additional values, such as IQE_ADDITIONAL_ENV_VARS='DYNACONF_ephemeral_images__aws__image_uuid=<image_uuid>"
```
Comment (ezr-ondrej) [inline tasks/run-iqe-cji.yaml:64]: I'd add `used when IQE_ENV_VARS are empty` just to have the intended behaviour described :)

**Comment by adarshdubey-star** [comment]
> no real comments, just a description improvement 👍 Nice job! Feel free to merge once ready 🎉

Updated all

**Pull request merged**